### ISSUE TITLE
Why does PeriodicPositionalEncoding have a dropout?

### ISSUE THREAD

**Issue by JeremyCJM** [EvelynFan/FaceFormer#47]
Hi Evelyn,

Why does PeriodicPositionalEncoding have a dropout?

Thanks,
Jeremy

**Comment by EvelynFan** [comment]
> Hi Evelyn,
> 
> Why does PeriodicPositionalEncoding have a dropout?
> 
> Thanks, Jeremy

Hi, this follows the default setting in the original position encoding of transformer. The class PositionalEncoding(nn.Module) has a dropout (https://github.com/pytorch/tutorials/blob/master/beginner_source/transformer_tutorial.py).
